cli/admin/src/test/java/io/pravega/cli/admin/dataRecovery/DataRecoveryTest.java

```diff
@@ -963,6 +969,58 @@ public void testTableSegmentRecoveryCommand() throws Exception {
         Assert.assertArrayEquals(Files.readAllBytes(Paths.get(pravegaStorageDir.toString(), listOfFiles.get(0).getName())), serializedEntries.getCopy());
     }
 
+    @Test
+    public void testTableSegmentRecoveryCommandUnVersioned() throws Exception {
```

Comment:
this is the versioned method right? name says unversioned

Reply:
Will update the name in some other pr.

Reply:
Well, that' very minor to create a PR for it. Not a big deal, we can change it in the next PR related to CLI.